test/integration/targets/module_utils/runme.sh

```diff
@@ -4,5 +4,36 @@ set -eux
 
 ANSIBLE_ROLES_PATH=../ ansible-playbook module_utils_basic_setcwd.yml -i ../../inventory "$@"
 
+cleanup() {
+    rm -fv /tmp/nulltest
+}
+
+trap cleanup EXIT
```

Comment:
Now that the test is using the output_dir this cleanup isn't needed is it?

Reply:
That is correct and I forgot to update the path in it anyway. :D